lib/actions/form.js

```diff
@@ -128,3 +130,50 @@ export function formChanged (oldQuery, newQuery) {
     }
   }
 }
+
+/**
+ * Check if the trip should be replanned based on the auto plan strategy,
+ * whether the mobile view is active, and the old/new queries. Response type is
+ * an object containing various booleans.
+ */
+export function checkShouldReplanTrip (autoPlan, isMobile, oldQuery, newQuery) {
```

Comment:
Might be nice to have some unit tests for this. This is non-blocking.

Reply:
Added a ticket for revisiting this here: https://github.com/opentripplanner/otp-react-redux/issues/355